filter.go

```diff
@@ -107,3 +110,18 @@ func ByGVK(gvk schema.GroupVersionKind) Predicate {
 		return u.GroupVersionKind() == gvk
 	}
 }
+
+// In returns true if present in the passed Manifest, using
+// "gvk|namespace/name" as a unique identifier
+func In(manifest Manifest) Predicate {
```

Comment:
This feels odd, shouldn't this take a GVK, namespace and name? The description says that this checks if a resource is in a manifest. This reads like it checks that the entire manifest is in another manifest. Am I missing something?

Reply:
Well, it's returning a `Predicate`, which is intended to filter *another* manifest. How would you suggest making it less odd?

Reply:
I read it wrong, it's fine I think.